src/components/button/Button.tsx

```diff
@@ -83,7 +116,8 @@ class Button extends React.Component<ButtonProps> {
                 type={type}
                 {...buttonProps}
             >
-                <span className="btn-content">{children}</span>
+                {textContent}
+                {iconContent}
```

Comment:
Do we need to solve for left/right positioning of the icon?

Reply:
No, non-standard ones will be migrated to right-side icons, except the special case dropdowns that have other styling for the padding and alignment anyways.